lib/phoenix/naming.ex

```diff
@@ -53,6 +53,25 @@ defmodule Phoenix.Naming do
   end
 
   @doc """
+  Inflects the default name of the PubSub server from the module name
+
+  ## Examples
+
+      iex> Phoenix.Naming.module_to_pub_server(MyApp.MyChannel)
+      MyApp.PubSub
+
+      iex> Phoenix.Naming.module_to_pub_server(MyApp.Admin.MyChannel)
+      MyApp.PubSub
+
+  """
+  def module_to_pub_server(mod) do
```

Comment:
Great idea, done in https://github.com/phoenixframework/phoenix/commit/7c485d910b6dbf230079fd0775504890a74d0431